acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/steps/site/SearchSteps.java

```diff
@@ -43,6 +44,30 @@ public void iSearchForThePublication() throws Throwable {
         iSearchFor(title);
     }
 
+    @Then("^I should see publication in search results$")
+    public void iShouldSeePublicationInSearchResults() throws Throwable {
+        String expectedTitle = testDataRepo.getCurrentPublication().getTitle();
+
+        List<String> actualResults = searchPage.getSearchResultWidgets()
+            .stream()
+            .map(SearchResultWidget::getTitle)
+            .collect(toList());
+
+        assertThat("Publication is in the results", actualResults, contains(expectedTitle));
+    }
+
+    @Then("^I should not see publication in search results$")
+    public void iShouldNotSeePublicationInSearchResults() throws Throwable {
+        String expectedTitle = testDataRepo.getCurrentPublication().getTitle();
+
+        List<String> actualResults = searchPage.getSearchResultWidgets()
+            .stream()
+            .map(SearchResultWidget::getTitle)
+            .collect(toList());
+
+        assertThat("Publication is in the results", actualResults, not(contains(expectedTitle)));
```

Comment:
Think this should be hasItem instead of contains. Contains says the list must exactly match rather than just have the item in it.